src/views/TenderDescription.vue

```diff
@@ -0,0 +1,155 @@
+    margin-left: 20%;">
+    <div class="tender-form-title">
+      <h1> Tender description </h1>
+    </div >
+      <h2>Contractor</h2>
+        <div class="rectangle" >
+      <div  >
+      <label>  Official Name: </label>
+        <div> {{this.tender.name}} </div>
+        <label class="form"> Country: </label>
+        <div > {{this.tender.countryId}} </div>
+        <label>National Registration Number: </label>
+        <div > {{this.tender.nationalRegNumber}} </div>
+        <label class="form">City/Town: </label>
+        <div > {{this.tender.townCity}} </div>
+      </div> </div>
+    <h3>Contact person</h3>
+        <div class="rectangle">
+      <label> Name: </label>
+      <div> {{this.tender.contactPersonName}} </div>
+      <label> Surname: </label>
+      <div> {{this.tender.contactPersonSurname}} </div>
+      <label> Phone Number:</label>
+      <div> {{this.tender.phoneNumber}} </div>
+    </div>
+
+    <h4>Procedure</h4>
+        <div class="rectangle">
+    <label> Procedure: </label>
+    <div> {{this.tender.name}} </div>
+    <label> Language: </label>
+    <div> {{"English"}} </div>
+        </div>
+      <h5>Subject matter of the procurement</h5>
+        <div class="bigRectangle">
+      <label> CPV Code: </label>
+      <div> {{this.tender.cvpCodeId}} </div>
+      <label> Type of tender:</label>
+      <div> {{this.tender.tenderTypeId}} </div>
+      <label>Min Tender Value: </label>
+      <div> {{this.tender.minValue}} </div>
+      <label>Max Tender Value: </label>
+      <div> {{this.tender.maxValue}} </div>
+      <label> Currency: </label>
+      <div> {{this.tender.currencyId}} </div>
+      <label> Description: </label>
+      <div> {{" "}} </div>
+    </div>
+
+      <h6> Date</h6>
+        <div class="rectangle">
+    <label> Publication Date: </label>
+    <div> {{this. formatDate(tender['publicDate'])}} </div>
+    <label> Deadline for Offer Submission: </label>
+    <div> {{this.formatDate(tender['deadLineOfSub'])}} </div>
+    <label> Deadline for Signed Contract Submission: </label>
+    <div> {{this.formatDate(tender['DeadForSinging'])}} </div>
+     </div>
+      <h7>Documents</h7>
+        <div>
+    </div>
+   </div>
+
+ </template>
+
+<script>
+export default {
+  name: "TenderDescription",
+  props:['tender'],
+  data() {
+      return {
+        name: null,
+        nationalRegNumber: null,
+        countryId: null,
+        townCity: null,
+        contactPersonName: null,
+        contactPersonSurname: null,
+        phoneNumber: null,
+        cvpCodeId: null,
+        tenderTypeId: null,
+        description: null,
+        minValue: null,
+        maxValue: null,
+        currencyId: null,
+        publicDate: null,
+        deadLineOfSub: null,
+        deadForSinging: null,
+
+      }
+      },
+  methods:{
+    formatDate(miliseconds) {
+      let d = new Date(miliseconds);
+      return ("0" + d.getDate()).slice(-2) + "-" + ("0" + (d.getMonth() + 1)).slice(-2) + "-" +
```

Comment:
this approach to format date is very wrong. Use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat or date-fns